examples/augmented-reality.html

```diff
@@ -71,6 +71,14 @@ <h3 style="margin-top: 40px; padding-bottom: 8px;" class="border-bottom">iOS, Ma
             <a href="https://developers.google.com/ar/develop/java/scene-viewer">Scene Viewer</a>
             on supported devices.</p>
           </li>
+          <li>
+            <div>ar-scale</div>
+            <p>Controls the scaling behavior in AR mode in
+            <a href="https://developers.google.com/ar/develop/java/scene-viewer">Scene Viewer</a>.
+            Set to "fixed" to disable scaling of the model, which sets it to
+            always be at 100% scale. Defaults to "auto" which allows the model to
+            be resized.</p>
+          </li>
```

Comment:
Do we need this here since it is also in index.html?

Reply:
Yeah, I think it's safe to omit the docs here in favor of the docs in `index.html` for now.